docs/source/language.rst

```diff
@@ -148,6 +148,20 @@ If you wish to make these text values available in English and French, you could
 In other words: each plain text value should be replaced by a dictionary, where the language code is the key and the translated text represents the matching value.
 For lists, this can be applied as well (see ``keywords`` example above), as long as you nest the entire list under a language key instead of each list item.
 
+A similar concept can be applied to the ``title-field`` property of the provider in a collection configuration. If a dataset contains multiple columns each representing the title 
+element in a specific language, you can configure the title-field accordingly.
+
+.. code-block:: yaml
+
+  providers:
+    - type: feature
+      name: GeoJSON
+      data: tests/data/ne_110m_lakes.geojson
+      title_field: 
+        en: name-en 
+        fr: name-fr
+        de: name-de
```

Comment:
Your example title-fields `name-en`, `name-fr`, and `name-de` were a bit confusing to me at first (I thought the title field had to use some locale label/construct).
Probably clearer to use `name_eng`, `name_fre`, `name_deu` or something completely different, so that it's clear that the use does not need to include language tags in the field name?